plugin/trino-openlineage/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>445-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
```

Comment:
Most of the `artifactId` are kept consistent to match the name of project directory inside `plugin`.

Should we change the artifactId to `trino-openlineage` instead ?

Reply:
sure!